lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -408,19 +408,26 @@ void SessionStateEnforcer::m5g_move_to_inactive_state(
 void SessionStateEnforcer::m5g_move_to_active_state(
     std::unique_ptr<SessionState>& session, SetSmNotificationContext notif,
     SessionStateUpdateCriteria* session_uc) {
-  bool get_gnb_teid = false;
-  bool get_upf_teid = false;
-
+  bool get_gnb_teid        = false;
+  bool get_upf_teid        = false;
+  uint32_t upf_teid        = 0;

```

Comment:
Why initialize upf_teid to 0 if we're assigning it later here?
```
upf_teid =
      update_session_rules(session, get_gnb_teid, get_upf_teid, session_uc);
```

Reply:
Yes, not required. Fixed it.